### DISCUSSION TITLE
External editor

### DISCUSSION THREAD

**Discussion by leventelist** [sqlitebrowser/sqlitebrowser#3246] [Q&A] [answered]
There's a button to edit a field with external program. Is there any way to explicitly set the external editor?

**Comment by chrisjlocke** [comment, marked as answer]
Not directly.  Depends on your OS (which you haven't said which one you use) but DB4S opens 'Notepad' for .txt files - so if you amend the default application for .txt files for something else, DB4S will open that application instead.

**Reply by leventelist** [reply]
I use Debian Linux. Sqlitebrowser opened the field with a schematic editor, which is not what I wanted.

**Reply by scottfurry** [reply]
This sounds like an issue with file mime-type association.
